pkg/base/replicated.go

```diff
@@ -202,28 +253,7 @@ func renderReplicatedHelmChart(kotsHelmChart *kotsv1beta1.HelmChart, upstreamFil
 	}
 	helmUpstream.Name = kotsHelmChart.GetDirName()
 
-	mergedValues := kotsHelmChart.Spec.Values
```

Comment:
we should put this back since it's not done anymore when the kots kinds are rendered, right?